brayns/common/scene/Scene.cpp

```diff
@@ -189,78 +194,103 @@ void Scene::buildDefault( )
     const Vector3f WHITE = { 1.f, 1.f, 1.f };
 
     // Cornell box
-    for( size_t material = 1; material < 6; ++material )
+    std::string engineName = _parametersManager.getRenderingParameters().getEngine();
+    if( engineName == "optix" && !getEmpty() )
     {
-        _materials[material]->setColor( colors[material] );
-        _materials[material]->setSpecularColor( WHITE );
-        _materials[material]->setSpecularExponent( 10.f );
-        for( size_t i = 0; i < 6; ++i )
+        const Vector3f BLACK = { 0.f, 0.f, 0.f };
+        for( size_t material = 0; material < 6; ++material )
```

Comment:
They do exist because 200 materials are created in the first place. I agree that materials should be dynamically created according to the requirement set by the geometry. To be done.